src/Provider/URLBuilder.js

```diff
@@ -102,11 +112,10 @@ export default {
         if (source.bboxDigits !== undefined) {
             precision = source.bboxDigits;
         }
-        bbox.as(source.crs, extent);
-        const w = extent.west.toFixed(precision);
-        const s = extent.south.toFixed(precision);
-        const e = extent.east.toFixed(precision);
-        const n = extent.north.toFixed(precision);
+        const w = bbox.west.toFixed(precision);
```

Comment:
there is no case where the bbox is not in the source crs already?

Reply:
I prefered to move the conversion before calling this function (cf first commit) to make it pure. As such `bbox` does one and one thing only and it is easier to track changes of crs.

Reply:
Ok, sorry I missed that. In this case, I think we should verify this assumption and either throw an error if it's not met or a log a warning and do the crs conversion; just to avoid future bug.